pkg/util/versionchecker/BUILD.bazel

```diff
@@ -0,0 +1,104 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+# Clone empty version of cert-manager repo and list all tags
+genrule(
+    name = "git_tags",
+    outs = [":git_tags.txt"],
+    cmd = "git ls-remote -t --refs https://github.com/jetstack/cert-manager.git | awk '{print $$2;}' | sed 's/refs\\/tags\\///' | sed -n '/v1.0.0-alpha.0/,$$p' > $@",
```

Comment:
It selects only the latest versions. Indeed it should instead start from 1.0.0 I'll change that.

Reply:
I only just understood how that `sed` command works. Had to re-read the man page. 
It filters tag names  starting at `v1.0.0.alpha.0` until the end of the output, right?

Maybe add that to a comment in the code so that the intent is clear.

```
Addresses
       Sed  commands can be given with no addresses, in which case the command will be executed for all input lines; with one address, in which case the command will only be executed for input lines which match that address; or with
       two addresses, in which case the command will be executed for all input lines which match the inclusive range of lines starting from the first address and continuing to the second address.  Three things to note about  address
       ranges:  the  syntax  is addr1,addr2 (i.e., the addresses are separated by a comma); the line which addr1 matched will always be accepted, even if addr2 selects an earlier line; and if addr2 is a regexp, it will not be tested
       against the line that addr1 matched.

```